ansible/tasks/stickypedia.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: "create stickypedia user"
+  user:
+    name: "stickypedia"
+    shell: "usr/sbin/nologin"
+    home: "/var/www/stickypedia"
+    system: true
+
+block:
+  - name: "clone git with Stickypedia content"
+    git:
+      repo: {{ stickypedia.git_content }}
```

Comment:
This also won't work, it'll need to be quoted :jinja_burning:

Reply:
fixed, thanks